transactions/add-keys/2023/nov-16/arguments_service_account.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "type": "Int",
+        "value": "17"
+    },
+    {
+        "type": "String",
+        "value": "475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be"
```

Comment:
this does not seem to be an encoded account key, which is what is expected by the tx, this is only the public key.

I believe the encoded key has to include the hash/sig algos and the weight:

`f846b840475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be020181fa`